tests/help.rs

```diff
@@ -841,7 +878,7 @@ fn no_wrap_help() {
     assert!(utils::compare_output(
         app,
         "ctest --help",
-        MULTI_SC_HELP,
+        &format!("{}\n", MULTI_SC_HELP),
```

Comment:
MULTI_SC_HELP already has a trailing newline, this adds a second one, which doesn't sound right

Reply:
On second thought, this is probably expected

Reply:
Can you explain why this is expected?

Reply:
Maybe we want to remove the extra new line if `override_help` has one trailing?